patches/minecraft/net/minecraft/client/MouseHelper.java.patch

```diff
@@ -1,71 +1,28 @@
 --- a/net/minecraft/client/MouseHelper.java
 +++ b/net/minecraft/client/MouseHelper.java
-@@ -71,6 +71,7 @@
-             this.field_198042_g = -1;
-          }
+@@ -18,6 +18,7 @@
+ @OnlyIn(Dist.CLIENT)
+ public class MouseHelper {
+    private final Minecraft field_198036_a;
++   public static final boolean IS_RUNNING_ON_MAC = net.minecraft.util.Util.func_110647_a() == net.minecraft.util.Util.OS.OSX;
```

Comment:
The Minecraft class already has a public constant for IS_RUNNING_ON_MAC, why not use that instead?

Reply:
Can that be linked from the package handler up top? Or does that require an import. I was told not to import, but, I will check and see it it does not need one.

Reply:
Yes, when I remove that variable definition, it denies it, although, let me try one other thing. ...

Reply:
If it's already imported I think it would work, I'm sure someone who knows better will correct me if I'm wrong

Reply:
Found out how, kind of new to a few things today.